bug in Building FPGA driver with AutoCounter config

### Background Work

- [X] Yes, I searched the [mailing list](https://groups.google.com/forum/#!forum/firesim)
- [X] Yes, I searched [prior issues](https://github.com/firesim/firesim/issues)
- [X] Yes, I searched the [documentation](https://docs.fires.im/en/stable/)

### FireSim Version and Hash

<!-- copy/paste the output of `git describe --tags --long` below -->
1.15.1-0-gf33e1f2f

### OS Setup

Linux ip-192-168-1-84.us-west-2.compute.internal 3.10.0-1160.66.1.el7.x86_64 #1 SMP Wed May 18 16:02:34 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: CentOS
Description:    CentOS Linux release 7.9.2009 (Core)
Release:        7.9.2009
Codename:       Core

### Other Setup

<!-- Ex: Prior steps taken / Documentation Followed / etc... -->
I tried to build custom hardware config which include WithAutoCounter

### Current Behavior

2022-11-14 21:45:22,673 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc: In constructor 'autocounter_t::autocounter_t(simif_t*, std::vector<std::__cxx11::basic_string<char> >&, AUTOCOUNTERBRIDGEMODULE_struct*, A
ddressMap, uint32_t, const char* const*, const uint32_t*, const uint32_t*, const uint32_t*, const uint32_t*, const char* const*, const char* const*, const char*, unsigned int, unsigned int, int)':
2022-11-14 21:45:22,673 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc:61:68: error: expected ')' before 'PRIu64'
2022-11-14 21:45:22,673 [flush       ] [DEBUG]  [localhost] out:    61 |              "[AutoCounter] Warning: requested sample rate of %" PRIu64
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:       |                                                                 ^~~~~~~
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:       |                                                                 )
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc:60:16: note: to match this '('
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:    60 |         fprintf(stderr,
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:       |                ^
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc:17:1: note: 'PRIu64' is defined in header '<cinttypes>'; did you forget to '#include <cinttypes>'?
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:    16 | #include <sys/mman.h>
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:   +++ |+#include <cinttypes>
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:    17 |
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc:61:66: warning: spurious trailing '%' in format [-Wformat=]
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:    61 |              "[AutoCounter] Warning: requested sample rate of %" PRIu64
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:       |                                                               ^
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out: /home/centos/firesim/sim/midas/src/main/cc/bridges/autocounter.cc:61:17: warning: too many arguments for format [-Wformat-extra-args]
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:    61 |                 "[AutoCounter] Warning: requested sample rate of %" PRIu64
2022-11-14 21:45:22,674 [flush       ] [DEBUG]  [localhost] out:       |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

### Expected Behavior

Build hareware with any bug

### Other Information

I include  #include \<cinttypes\> in autocounter.cc  and there is no bug anymore.

--- a/sim/midas/src/main/cc/bridges/autocounter.cc
+++ b/sim/midas/src/main/cc/bridges/autocounter.cc
@@ -7,6 +7,7 @@
 #include <regex>
 #include <stdio.h>
 #include <string.h>
+#include <cinttypes>
 
 #include <fcntl.h>
 #include <sys/stat.h>

--- a/sim/midas/src/main/cc/bridges/autocounter.cc
+++ b/sim/midas/src/main/cc/bridges/autocounter.cc
@@ -2,12 +2,12 @@
 
 #include "autocounter.h"
 
+#include <cinttypes>
 #include <iostream>
 #include <limits.h>
 #include <regex>
 #include <stdio.h>
 #include <string.h>
-#include <cinttypes>
 
 #include <fcntl.h>
 #include <sys/stat.h>

